### DISCUSSION TITLE
Does the new version come with undo/redo functionality?

### DISCUSSION THREAD

**Discussion by cristy972** [Blazor-Diagrams/Blazor.Diagrams#314] [General]
Will support the diagram an undo function? Or will be a way that in changed action to get a parameter to get what triggered it?

**Comment by zHaytam** [comment]
Hello, unfortunately, no, the new version doesn't come with undo/redo.  
That functionality is a pretty big one and I'm not ready yet to tackle it sorry!

That being said, there are a lot of events in the library that can help you figure out exactly what changed. The Changed event is only for refreshes to the UI (mostly).

**Reply by cristy972** [reply]
Ok, thank you for the quick response :D
